src/Symfony/Component/HttpKernel/Tests/HttpCache/HttpCacheTest.php

```diff
@@ -2070,6 +2070,26 @@ public function testTraceLevelShort()
         $this->assertTrue($this->response->headers->has('X-Symfony-Cache'));
         $this->assertEquals('miss', $this->response->headers->get('X-Symfony-Cache'));
     }
+
+    public function testQueryRequestAreCacheable()
+    {
+        $this->setNextResponse(200, ['Cache-Control' => 'public, s-maxage=3600']);
+        $this->request('QUERY', content: '{"foo": "bar"}');
```

Comment:
using named parameters in tests is a no-go. Tests should use the API covered by our BC promise, as that helps detecting unintentional BC breaks in some cases (if we have to make unexpected changes in tests so that they work). And parameter names are not covered by our BC promise in Symfony (except for attribute constructors)